STYLEGUIDE.md

```diff
@@ -168,11 +168,13 @@ window.clearTimeout(1);
 document.createElement('div');
 ```
 
-### Use `useIsomorphicLayoutEffect` instead of `useLayoutEffect`
+### Import `useIsomorphicLayoutEffect as useLayoutEffect` instead of using `React.useLayoutEffect`
 
 ```jsx
 // Good
-useIsomorphicLayoutEffect(() => {});
+import {useIsomorphicLayoutEffect as useLayoutEffect} from '../utils/index.js'
+
+useLayoutEffect(() => {});
 ```
 
 ```jsx
```

Comment:
The bad examples could also include an example of using `useIsomorphicLayoutEffect` directly.

Reply:
The name of the export has changed to `useLayoutEffect`, so using `useIsomorphicLayoutEffect` directly shouldn't be possible anymore.